src/ports/postgres/modules/validation/internal/cross_validation.py_in

```diff
@@ -336,22 +339,31 @@ class CrossValidator(object):
                                        dependent_varname=self._target_col,
                                        independent_varname=self._features_col,
                                        **sub_args)
+        plpy.info("===================================================")
+        plpy.info("{0}".format(str(args)))
+        plpy.info("===================================================")
+
         estimator(**args)
 
+        plpy.info("++++++++++++++++++++++++++++++++++++++++++++")
+
         schema_madlib = args['schema_madlib']
         output_table = unique_string(desp='output_table')
         model_table = args['model_table']
 
-        predictor(schema_madlib, model_table,
-                  rel_valid, col_id, output_table)
+        model_vals = plpy.execute("SELECT * FROM " + model_table)
+        plpy.info("---- Model table {0} = {1}".format(model_table, str(model_vals)))
+        data_cols = get_cols_and_types(rel_valid)
+        plpy.info("---- Source table columns = " + str(data_cols))
+
+        plpy.info("--------------------------------------------")
+        predictor(schema_madlib, model_table, rel_valid, col_id, output_table)
 
```

Comment:
The various `plpy.info()` calls are flooding the interface with lots of messages. Do we want to print out these info strings all the time?